drupal/sites/default/boinc/modules/boincwork/boincwork.module

```diff
@@ -162,6 +162,32 @@ function boincwork_menu() {
     'access arguments' => array('access content'),
     'type' => MENU_CALLBACK
   );
+  // @todo - change URL of routes
```

Comment:
No. I changed the URL paths but did not remove the comment. I have now removed the comment.